core/modules/wmgr/python/dbMgr.py

```diff
@@ -45,10 +45,16 @@
 from .errors import errorResponse, ExceptionResponse
 from flask import Blueprint, json, request, url_for
 from lsst.db import db
+from lsst.db.mysqlUtils import *
+from lsst.db.mysqlUtils import createDb as createDb_
+from lsst.db.mysqlUtils import dropDb as dropDb_
+from lsst.db.mysqlUtils import listTables as listTables_
```

Comment:
`import *` is problematic in general, so it's better to replace 4 lines above with just `from lsst.db import mysqlUtils` and use `mysqlUtils.something()` instead